Exploration of the MNIST1000 Dataset.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>I didn't get a chance to explain this, but we typically keep all development inside the&nbsp;<code>project</code>&nbsp;folder (feel free to rename it). This will make more sense when we set up our&nbsp;<code>Dockerfile</code></p><p> </p><p><code>Also, can you add these libraries to <code>requirements.txt</code>?</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>